src/Pages/LoginPage/style.ts

```diff
@@ -85,11 +85,11 @@ export const SignupText = styled.p`
   margin-right: 5px;
 `;
 
-export const PasswrdContainer = styled.div`
+export const PasswordContainer = styled.div`
   display: flex;
 `;
 
-export const PasswrdToggleButton = styled.div`
+export const PasswordToggleButton = styled.div`
```

Comment:
```suggestion
export const PasswordToggleButton = styled.button`
```

버튼의 역할을 하는 컴포넌트로 보입니다